bin/create-release

```diff
@@ -0,0 +1,200 @@
+#! /usr/bin/env python3
+
+import os
+import re
+import requests
+import subprocess
+import shlex
+import sys
+
+# DOC: Create a new draft CiviForm image release. Usage: bin/create-release COMMIT_SHA RELEASE_NUMBER
+#
+# Requirements:
+#   Must have creds to login to docker hub as 'civiform' user.
+#   Must have a GitHub API token with repo and read:org scopes located at 'gh-release-token.txt'
+#
+# When successful, this script:
+#   1. tags the git commit for the release with the release number
+#   2. tags the docker image for the release with the release number
+#   3. creates a new draft release on github
+
+if not os.path.exists('gh-release-token.txt'):
+    print(
+        '''
+        gh-release-token.txt not found in root directory
+        Create one with repo and read:org scopes at https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
+    ''',
+        file=sys.stderr)
+    exit(1)
+
+if len(sys.argv) != 3:
+    print(
+        'Usage: bin/create-release COMMIT_SHA RELEASE_NUMBER', file=sys.stderr)
+    exit(1)
+
+
+def shell_cmd(command_string):
```

Comment:
Please create a main function, so that we're not defining functions inside the flow of logic.

```
def main():
    print("Hello World!")

if __name__ == "__main__":
    main()
```
https://google.github.io/styleguide/pyguide.html#317-main

Reply:
done